docs/src/concepts/fine-tuning.rst

```diff
@@ -0,0 +1,217 @@
+.. _label_fine_tuning_concept:
+
+Fine-tune a pre-trained model
+=============================
+
+.. warning::
+
+  This section of the documentation is only relevant for PET model so far.
+
+This section describes the process of fine-tuning a pre-trained model to
+adapt it to new tasks or datasets. Fine-tuning is a common technique used
+in machine learning, where a model is trained on a large dataset and then
+fine-tuned on a smaller dataset to improve its performance on specific tasks.
+So far the fine-tuning capabilities are only available for PET model.
+
+There is a complete example in :ref:`Fine-tune example <fine-tuning-example>`.
+
+.. note::
+
+  Please note that the fine-tuning recommendations in this section are not universal
+  and require testing on your specific dataset to achieve the best results. You might
+  need to experiment with different fine-tuning strategies depending on your needs.
+
+
+Basic Fine-tuning
+-----------------
+
+The basic way to fine-tune a model is to use the ``mtt train`` command with the
+available pre-trained model defined in an ``options.yaml`` file. In this case, all the
+weights of the model will be adapted to the new dataset. In contrast to the
+training continuation, the optimizer and scheduler state will be reset. You can still
+adjust the training hyperparameters in the ``options.yaml`` file, but the model
+architecture will be taken from the checkpoint.
+To set the path to the pre-trained model checkpoint, you need to specify the
+``read_from`` parameter in the ``options.yaml`` file:
+
+.. code-block:: yaml
+
+  architecture:
+    training:
+      finetune:
+        method: "full" # This stands for the full fine-tuning
+        read_from: path/to/checkpoint.ckpt
+
+We recommend to use a lower learning rate than the one used for the original training,
+as this will help stabilizing the training process. I.e. if the default learning rate is
+``1e-4``, you can set it to ``1e-5`` or even lower, using the following in the
+``options.yaml`` file:
+
+.. code-block:: yaml
+
+  architecture:
+    training:
+      learning_rate: 1e-5
+
+Please note, that in most use cases you should invoke a new energy head by specifying
+a new energy variant. The variant naming follows the simple pattern
+``energy/{variantname}``. A reasonable name could be the energy functional or level of
+theory your dataset was trained on, e.g. ``energy/pbe``, ``energy/SCAN`` or even
+``energy/dataset1``. Further you can add a short description for the new variant, that
+you can specify in ``description`` of your ``options.yaml`` file.
```

Comment:
I think you can link to the description documentation page using `` :attr:`description` ``. You may have to add the specific modulename. @pfebrer probably know better 😅

```suggestion
``energy/dataset1``. Further we recommend adding a short :attr:`description` for the new variant, that
you can specify in ``description`` of your ``options.yaml`` file.
```